apis/swagger.yml

```diff
@@ -58,6 +58,17 @@ paths:
         500:
           $ref: "#/responses/500ErrorResponse"
 
+  /metrics:
+    get:
+      summary: "Get Prometheus metrics"
+      description: "Get Prometheus metrics"
+      responses:
+        200:
```

Comment:
I am wondering if there is any possibilities that the endpoint returns a non-200 status code?

Reply:
I think it is always 200. It is just a Get request without any  parameters. If this /metrics cannot work, the application will throw an error(maybe duplicate of metrics names or bad metric help), rather than returning non-200 status code.